open_spiel/bots/stockfish/.gitignore

```diff
@@ -0,0 +1,2 @@
+# This directory is fetched during install, do not include in OpenSpiel.
```

Comment:
I know it may seem a detail, but because of the way we sync between Google and Github, it's easier for us if we regroup all ignore into https://github.com/deepmind/open_spiel/blob/master/.gitignore

As you can see, there is a " # External git modules" section.

Could you please add any .gitignore into this file?